index.js

```diff
@@ -5,6 +5,7 @@ const isWsl = require('is-wsl');
 const chalk = require('chalk');
 const child_process = require('child_process')
 const version = require('./package.json').version;
+__dirname = __dirname.replace(/ /g,"\\\ ");
 const virtcrudVersion = JSON.parse(child_process.execSync(`cd ${__dirname} && npm list virtcrud --json`).toString()).dependencies.virtcrud.resolved.split('#')[1].substring(0, 7);
```

Comment:
```suggestion
 const virtcrudVersion = JSON.parse(child_process.execSync(`npm list virtcrud --json`, {cwd: __dirname}).toString()).dependencies.virtcrud.resolved.split('#')[1].substring(0, 7);

```

Reply:
A better fix would involve adjusting passing the cwd. Which should also escape spaces.